src/main/java/svc/managers/ViolationManager.java

```diff
@@ -22,4 +22,12 @@ public Violation GetViolationById(int violationId) {
 		return violationDAO.getViolationsByCitationNumber(citationNumber);
 	}
 	
+	public boolean insertDemoViolations(List<Violation> violations){
```

Comment:
Same thing here on naming...I'd omit "Demo" from here. This thing is inherently demo.